### DISCUSSION TITLE
Error when run the bot

### DISCUSSION THREAD

**Discussion by gsalasan** [conor19w/Binance-Futures-Trading-Bot#11] [Q&A] [answered]
Hi Conor, 

'NoneType' object is not subscriptable
<class 'TypeError'> Bot.py 183

I got this error, what it's mean?

**Comment by conor19w** [comment, marked as answer]
Hey Man,
Are you on the most up to date version? This error cropped up for me a day or two ago I think I fixed it though.

**Reply by conor19w** [reply]
Sorry I forgot to update the script, just did it there should be working now. Let me know if not

**Reply by gsalasan** [reply]
done pulling and got new error

name 'copy' is not defined
<class 'NameError'> Bot.py 50

**Reply by conor19w** [reply]
Just missing an import, try again now

**Comment by gsalasan** [comment]
Bot running and order placed, after remove manualy BZRXUSDT that coz this 

Symbol: BZRXUSDT, error: APIError(code=-4141): Symbol is closed.

after order placed, there is another show up 

list index out of range
<class 'IndexError'> Bot.py 185

**Reply by conor19w** [reply]
Can you give me a readout from the console exactly?

**Reply by conor19w** [reply]
Did an order place?

**Reply by gsalasan** [reply]
Starting FILUSDT web socket
Starting STMXUSDT web socket
Starting BALUSDT web socket
Starting GALAUSDT web socket
Starting BATUSDT web socket
Starting AKROUSDT web socket
Starting XMRUSDT web socket
Starting COTIUSDT web socket
Setting Leverage...
Combining Historical and web socket data...
Finished.
Start Balance: 
2021-12-20 15:31:00: Attempting to place order on RAYUSDT
Initial Order Successful on RAYUSDT
2021-12-20 15:31:00: Trade Finished
Account Balance: 
list index out of range
<class 'IndexError'> Bot.py 185
2021-12-20 15:32:00: Attempting to place order on RAYUSDT
Initial Order Successful on RAYUSDT
2021-12-20 15:32:00: Trade Finished
Account Balance: 
list index out of range
<class 'IndexError'> Bot.py 185
2021-12-20 15:33:00: Attempting to place order on RAYUSDT

I can't upload the screen shot, but that ^ exact output from console , minus the balance 

yes the order is placed, but gone in a sec.

**Reply by conor19w** [reply]
Okay So no Order gets executed? It just gets instantly cancelled?

**Reply by gsalasan** [reply]
yes

**Reply by conor19w** [reply]
Okay Should work now sorry about that, I've a few bots with similar templates and forgot to update the public one, been dealing with these errors the past few days.

**Reply by mujahidazamcsm** [reply]
Hi @conor19w - could u plz confirm if we have all the latest code available.. or anything minor still missing?

**Reply by conor19w** [reply]
Yeah the code is up to date, Why what's the issue?

**Reply by mujahidazamcsm** [reply]
> Yeah the code is up to date, Why what's the issue?

No issues.. just checking to be sure after seeing this discussion
